### DISCUSSION TITLE
Export Avatar

### DISCUSSION THREAD

**Discussion by joeblau** [family/connectkit#18] [Ideas]
Could you please Export the code used to generate the `Avatar`/`EnsAvatar`? I'm building my own custom button and I want to have an Avatar that is consistent with the Avatar generated in the modal.

Thanks!

**Comment by lochie** [comment]
Hello @joeblau,
This is definitely something on our roadmap, and we're hopefully going to get this feature out within the next few releases, we just have a few tweaks we need to do before we feel like it's fully extensible and reliable enough for devs to use in their dApps.

Will leave a follow up message in this thread when this feature is ready 😊

**Reply by joeblau** [reply]
Awesome! This library is absolutely amazing.

**Comment by lochie** [comment]
@joeblau we now export the avatar in version `1.0.0`!
